ubuntudesign/design.ubuntu.com
# Remove filtering from examples page

There are 20 examples. They can all fit on your screen at once. I feel that filtering is overkill.

And not just because it is easier.

QA
---

``` bash
bundle install
bundle exec jekyll serve
```

Go to <http://127.0.0.1:4000/examples>, see that it is absolutely fine that way. Move on with your life. Don't even look at <http://design.ubuntu.com/examples>.
